### ISSUE TITLE
Batch Normalization not Dropped

### ISSUE THREAD

**Issue by ylfzr** [flrngel/cpc-tensorflow#3]
Thanks very much for this share.
I noticed that the original paper said they do not use batch normalizaiton in the ResNet101 encoder, however,  bn is still applied in this implementation. Will this be problematic?  How will this change affects the representations extracted by the model?

**Comment by flrngel** [comment]
I missed batch norm during massive code changes. I'll check on this issue in 24 hour and will mention you again. Thanks!

**11.08**
still have no time.. wait for extra 24 hour please 😭

**Comment by flrngel** [comment]
I've update code which doesn't use batch normalization.
Thanks!
